### ISSUE TITLE
"dvh" units are only supported in recent browsers 

### ISSUE THREAD

**Issue by Stef2k16** [mantinedev/mantine#4622]
### What package has an issue

@mantine/core

### Describe the bug

In #4517 , `vh` units have been replaced with `dvh` units. This change can lead to an incorrect height of modals in older, but - according to the documentation - still supported browsers.

According to [the Mantine documentation](https://mantine.dev/pages/about/), for example Chrome is supported in version 84 and newer. However, the `dvh` height unit has only been introduced in chrome 108. For both Firefox and Safari, `dvh` are also only supported in more recent versions.

### What version of @mantine/hooks page do you have in package.json?

6.0.17

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

### Possible fix

- Use a different height unit
- Adjust the minimal supported browser versions

**Comment by 7iomka** [comment]
with css it is possible to pass two prop:value, but with emotion this is impossible without hacks like double className
```ts

el: {
height: '100vh',
},
elModern: {
height: '100dvh'
}

cx(classes.el, classes.elModern)
```
I don't know if this trick will get the same result that we have in css with fallback rule, and modern one after it.

**Comment by rtivital** [comment]
This issue is closed for one of these reasons:

- It was fixed in 7.0
- It is no longer reproducible in 7.0
- It is no longer applicable in 7.0
- It does not have a reproduction

If you think that this issue was closed by mistake and it is still an issue in version 7.0, please reopen it.
